src/stores/pages.js

```diff
@@ -444,14 +443,14 @@ export const usePagesStore = defineStore('pages', {
 
 		toggleCollapsed(pageId) {
 			// Default to 'false' if unset
-			set(this.collapsed, pageId, this.collapsed[pageId] == null ? false : !this.collapsed[pageId])
+			set(this.collapsed, pageId, this.collapsed[pageId] === undefined ? false : !this.collapsed[pageId])
 		},
 		collapse(pageId) { set(this.collapsed, pageId, true) },
 		expand(pageId) { set(this.collapsed, pageId, false) },
 
 		toggleOutline(pageId) {
 			// Default to 'true' if unset
-			set(this.outline, pageId, this.outline[pageId] == null ? true : !this.outline[pageId])
+			set(this.outline, pageId, this.outline[pageId] === undefined ? true : !this.outline[pageId])
```

Comment:
Is it `boolean | undefined`?

Reply:
Yes, either the array key doesn't exist yet, or it's a boolean.